### ISSUE TITLE
Training MSFC-NET on HBB annotations dataset

### ISSUE THREAD

**Issue by TerrafYassin** [ZhAnGToNG1/MSFC-Net#3]
Can i use your model to traind dataset annotated with HBB not OBB without making any change in your code ?

**Comment by ZhAnGToNG1** [comment]
Yes. you can use this code to train other datasets annotated with HBB. But you need to define your dataset like [https://github.com/ZhAnGToNG1/MSFC-Net/blob/master/src/lib/datasets/dataset/DOTA.py](url) and add your dataset into [https://github.com/ZhAnGToNG1/MSFC-Net/blob/master/src/lib/datasets/dataset_factory.py#L3](url).

**Comment by TerrafYassin** [comment]
please ,  i  want to train NWPU VHR 10 dataset using your model , need i keep the same code in DOTA.py changing just paths and import NWPU VHR in dataset_factory.py  or i need to do some changes to train my dataset ?
